llvm/include/llvm/MC/MCParser/MCTargetAsmParser.h

```diff
@@ -439,14 +439,14 @@ class MCTargetAsmParser : public MCAsmParserExtension {
   /// \param Operands [out] - The list of parsed operands, this returns
   ///        ownership of them to the caller.
   /// \return True on failure.
-  virtual bool ParseInstruction(ParseInstructionInfo &Info, StringRef Name,
+  virtual bool parseInstruction(ParseInstructionInfo &Info, StringRef Name,
```

Comment:
The comment needs updating.

Reply:
By just removing the reference to the function name, it's not necessary anymore